sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -44,6 +44,8 @@ export enum AppRoutes {
   SearchInstitutions = "search-institutions",
   InstitutionDetail = "institution-detail/:institutionId",
   Profile = "profile",
+  PendingDesignations = "institution/designation/pending",
+  DesignationAESTView = "institution/designation/view/:designationAgreementId",
```

Comment:
shouldn't it be `"institution/designation/designation-agreement/:designationAgreementId`?

Reply:
designation and designation-agreement both are same thing functionally, so did just kept only designation.

Reply:
or `"institution/designation-agreement/:designationAgreementId`

Reply:
changed the param name to designationId as discussed.